libs/coin-bitcoin/src/explorer.ts

```diff
@@ -0,0 +1,40 @@
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import invariant from "invariant";
+import { getEnv } from "@ledgerhq/live-env";
+
+type LedgerExplorer = {
+  version: string;
+  id: string;
+  endpoint: string;
+};
+
+const findCurrencyExplorer = (currency: CryptoCurrency): LedgerExplorer | null | undefined => {
+  if (getEnv("SATSTACK") && currency.id === "bitcoin") {
```

Comment:
not directly related to the PR. it is a tech debt.
@haammar-ledger as we can't support SATSTACK since v4. 
Should we remove the code related to SATSTACK then?